infra/terraform/terraform_test.go

```diff
@@ -0,0 +1,97 @@
+package terraform
+
+import (
+	"encoding/json"
+	"github.com/gravitational/robotest/infra/providers/gce"
```

Comment:
nit: this should be in a separate import group

Reply:
fixed in 8eabb37.  Down the road I'd like to see if I can implement this as a lint rule/guideline.  I'm kinda surprised gofmt/goimports doesn't automatically take care of this.

Reply:
It's a good question - which one are you using and how (i.e. directly or with editor/IDE integration)?

Reply:
I use vim-go, which I just learned runs at 'vanilla' gofmt upon buffer save.  I checked `goimports` now, and it fixes this issue.  I'll fix my vimrc to change the default to goimports. :facepalm: 

Regardless of personal editor setups, I'd like our `make lint` target to catch nits like this. Once https://github.com/gravitational/robotest/issues/171 hits then we can largely save human reviewers the time, and encourage folks like me to configure their editors the right way.

I'll look at what it would take to do that.